matminer/featurizers/tests/test_structure.py

```diff
@@ -304,6 +305,23 @@ def test_op_structure_fingerprint(self):
         for i in range(52, len(opvals)):
             self.assertAlmostEqual(int(opvals[i] * 500 + 0.5), 0)
 
+    def test_ewald(self):
+        # Add oxidation states to all of the structures
+        for s in [self.nacl, self.cscl, self.diamond]:
+            s.add_oxidation_state_by_guess()
+
+        # Test basic
+        ewald = EwaldEnergy(accuracy=2)
+        self.assertArrayAlmostEqual(ewald.featurize(self.diamond), [0])
+        self.assertLess(ewald.featurize(self.nacl)[0], 0)
```

Comment:
Can you test the actual value rather than just <0? 2 decimal places via assertAlmostEqual would be fine